DQMOffline/EGamma/python/photonAnalyzer_cfi.py

```diff
@@ -12,6 +12,8 @@
 
     barrelRecHitProducer = cms.InputTag('reducedEcalRecHitsEB'),								
     endcapRecHitProducer = cms.InputTag('reducedEcalRecHitsEE'),
+    barrelRecHitCollection = cms.InputTag (''),
```

Comment:
@NiharSaha why these additions?

Reply:
`barrelRecHitCollection` and `endcapRecHitCollection` both are in cosmicPhotonAnalyzer_cfi.py, but were not defined photonAnalyzer_cfi.py from where they were imported. Please correct me if I am wrong!

Reply:
I believe they can be dropped in both files since the parameter indeed does not exist in the class:

https://github.com/cms-sw/cmssw/blob/37bf1fa981cca07a2ee330cf20196954dbb5f707/DQMOffline/EGamma/plugins/PhotonAnalyzer.cc#L20-L46

The *RecHitProducer are used instead